README.md

```diff
@@ -47,8 +47,12 @@ If you just want the Python library, use: `pip install coinkite-tap-protocol`
 3. Run:
 
 ```
-pip install -r requirements.txt
```

Comment:
do not install from requirements.txt as there is no proper separation between test and lib and lib+cli